### PR TITLE
Create a documented issue template for bugs

### PR DESCRIPTION
**Pull request by securechicken**
This should ensure proper and complete information for bugs opening, as well as some sort of maniac-filter.
Next in anti-social-networks-confusion plan, templates to request enhancement/features, template to just ask questions and discuss because Facebook is offline, template to inform, and template to just get feedback before implementing something.

Use the "view file" on the created file to see the rendered result.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+### Environment
+<!-- Write your information on the right column between the | | characters. Do not delete/modify any | character. If you need to use a | character in your writing, or code, put them between back-quotes, `like this`. -->
+|Item|Your information|
+|---|---|
+|**Harware**|<!-- Version of your hardware: RevE Rebboted, RevG, Other, I don't known. If you don't know, it's probably written on it under the Chameleon picture. As a reminder, this repositoty only supports RevE Rebooted for now -->
+|**Buying source**|<!-- Put a URL to the merchant website where you  bought your ChameleonMini, or write a description of it. -->|
+|**Firmware**|<!-- Write the full version name of your firmware. Use the "VERSION" command, or look at "Settings" tab in GUI. -->|
+|**GUI**|<!-- Put a URL to file or repository source whre you found the GUI you are using to program or configure your ChameleonMini. If you do not use one and use commands only, type Terminal. As a reminder, rebootedGUI only is supposed to work with firmware in this repo. -->|
```
Comment (iceman1001) [inline .github/ISSUE_TEMPLATE/bug.md]: Not true,   RebootedGUI supports both RevE and RevG

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+|**Flashing environment**|<!-- Briefly describe the environment from which you compile, flash and program your CHameleonMini (OS, version). If you use different ones from different activities, or could reproduce the bug on different environments, describe them all. If it is not applicable, put N/A. -->|
+|**Flashing interface**|<!-- How do you connect your ChameleonMini to your computer to flash it. Please specify amongst: USB, AVRISPmkII, other (describe). If it is not applicable, put N/A. -->|
+|**Flashing method**|<!-- How do you flash a firmware to your ChameleonMini. Please specify amongst: dfu-programmer, flash.bat, BOOT_LOADER_EXE.exe, avrdude, other (describe). If it is not applicable, put N/A. -->|
+|**Flash memory space**|<!-- How many flash memory space do you have in your device. Find this information with "SPI_FLASHINFO" command. -->|
+|**Makefile configuration**|<!-- Tell which configuration/settings you tuned in Makefile. If it is not applicable, put N/A. -->|
+### About my use
+<!-- Please briefly describe how/why you mainly use a ChameleonMini, and what you are trying to do with it. -->
```
Comment (iceman1001) [inline .github/ISSUE_TEMPLATE/bug.md]: `For a bug report you really pushing it...

File: .github/ISSUE_TEMPLATE/bug.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+### About me
+<!-- Put an "X" for each item you achieved or agree with, between [ ], like this: [X]. If you do not know, do not check. -->
+* [ ] I have knowledge on RFID/NFC standards and/or technologies
+* [ ] I know how to program (<!-- Please detail relevant langages that are used within this repo if you want: C, Bash, Python, etc. -->)
+* [ ] I have knowledge on AVR microcontrollers programming
+* [ ] I know how to do a Pull Request (PR), or am OK to learn it and try it
+
```
Comment (iceman1001) [inline .github/ISSUE_TEMPLATE/bug.md:26]: Fun... but no.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+## Bug description
+### Situation
+<!-- Please describe what you are trying to do when the bug appears, and in which situation. Please also describe the 2 or 3 last things you did with your ChameleonMini before you were in the situation where the bug appeared, if possible. -->
+### Expected function and references
+<!-- Please describe how things are supposed to work from your perspective, and give URL or paths to this repository or Wiki pages that actually tell this is how it is supposed to happen. -->
```
Comment (iceman1001) [inline .github/ISSUE_TEMPLATE/bug.md]: Yet again fun,  but no.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+<!-- If you do not understand what is requested somewhere in here, please read Wiki pages and read other issues first. -->
+## Context
+### Prerequisites
+<!-- Put an "X" for each item you achieved or agree with, between [ ], like this: [X]. If you do not know, do not check. You are most probably not ready to submit a bug report if not all items are checked, or the bug report should not be submitted on this repository: if you do so anyway, you may receive no feedback. -->
+* [ ] I can reproduce the issue/bug that I report on
+* [ ] I run the latest version of the firmware from this repository
+* [ ] I am available to provide further details, and can reproduce the same initial situation where I identified the issue
+* [ ] I checked all available open sources, like Wiki, Internet search engines, and other issues, to document what I found in the better possible extent, confirm that something did not function as expected, and that a bug fix is required in this repository code
```
Comment (iceman1001) [inline .github/ISSUE_TEMPLATE/bug.md]: available?  haha,   if the issue gets old,  we just close it.

File: .github/ISSUE_TEMPLATE/bug.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+* [ ] I know how to do a Pull Request (PR), or am OK to learn it and try it
+
+## Bug description
+### Situation
+<!-- Please describe what you are trying to do when the bug appears, and in which situation. Please also describe the 2 or 3 last things you did with your ChameleonMini before you were in the situation where the bug appeared, if possible. -->
+### Expected function and references
+<!-- Please describe how things are supposed to work from your perspective, and give URL or paths to this repository or Wiki pages that actually tell this is how it is supposed to happen. -->
+### Bug
+<!-- Please describe the bug you identified, i.e. in what ways it did not function as expected. -->
+<!-- Give as much details as possible. Include commands, terminal output, and/or code lines where relevant, between back-quotes `like this` for a simple line, or between back-quotes paragraphs for multiple lines ``` LIKE THIS ```. -->
```
Comment (iceman1001) [inline .github/ISSUE_TEMPLATE/bug.md:32]: Agreed

**Review by iceman1001 — CHANGES_REQUESTED** [review]
Good initative,   but you are too harsh and strict.   Lighten up.

File: .github/ISSUE_TEMPLATE/bug.md (hunk 1 above)
Comment (securechicken) [reply .github/ISSUE_TEMPLATE/bug.md]: My bad (English), this is not what I wanted to tell, but rather the opposite. On my way.

File: .github/ISSUE_TEMPLATE/bug.md (hunk 4 above)
Comment (securechicken) [reply .github/ISSUE_TEMPLATE/bug.md]: What's fun and no?

Comment (iceman1001) [reply .github/ISSUE_TEMPLATE/bug.md]: the whole paragraph "expected function and references"

**Review by iceman1001 — APPROVED** [review]

**Pull request merged**